marimo/_utils/formatter.py

```diff
@@ -43,7 +44,7 @@ def format(self, codes: CellCodes) -> CellCodes:
 class RuffFormatter(Formatter):
     def format(self, codes: CellCodes) -> CellCodes:
         try:
-            process = subprocess.run("ruff", capture_output=True)
+            process = subprocess.run(f"{sys.prefix}/ruff", capture_output=True)
```

Comment:
I am not confident this will work reliably. 

More reliable might be `subprocess.run(["python", "-m", "ruff"], capture_output=True)`

Reply:
it looks like, the way tests are written, there is in fact no ruff right next to marimo; therefore I'm changing to first check if it exists there, if not, fall back to what it was before (just "ruff")